src/main/java/data/plans/PlanList.java

```diff
@@ -1,4 +1,140 @@
 package data.plans;
 
+import commands.PlanCommand;
+import data.workouts.Workout;
+import data.workouts.WorkoutList;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
 public class PlanList {
+    private WorkoutList workoutList;
+    private HashMap<String, Plan> plansHashMapList = new HashMap<>();
+    private ArrayList<String> plansDisplayList = new ArrayList<>();
+
+    public PlanList(WorkoutList workoutList) {
+        this.workoutList = workoutList;
+    }
+
+    public ArrayList<String> getPlansDisplayList() {
+        return this.plansDisplayList;
+    }
+
+    public HashMap<String, Plan> getPlansHashMapList() {
+        return this.plansHashMapList;
+    }
+
+    public static final int MAX_NUMBER_OF_WORKOUTS_IN_A_PLAN = 10;
```

Comment:
Perhaps this constant can be moved to the top of the class so as to maintain consistency with the other classes.